.gitignore

```diff
@@ -6,4 +6,5 @@ sb_values/
 src/asmcnc/production/database/credentials.py
 src/asmcnc/production/z_head_mechanics_jig/z_head_mechanics_jig_graph.png
 src/asmcnc/production/database/csvs/
-src/asmcnc/apps/systemTools_app/screens/calibration/sg_value_plots.png
\ No newline at end of file
+src/asmcnc/apps/systemTools_app/screens/calibration/sg_value_plots.png
+src/asmcnc/apps/systemTools_app/screens/xy_jig/xy_jig_graph.png
```

Comment:
if you have different graph names for forwards and away make sure this is updated too :)

Reply:
Addressed in another comment